server/routes/index.ts

```diff
@@ -59,30 +82,30 @@ export function defineRoutes(router: IRouter, securityConfigClient: IClusterClie
   function validateRequestBody(resourceName: string, requestBody: any): any {
     let inputSchema;
     switch (resourceName) {
-      case 'internalusers': inputSchema = internalUserSchema; break;
-      case 'actiongroups': inputSchema = accountSchema; break;
-      case 'rolesmapping': inputSchema = roleMappingSchema; break;
-      case 'roles': inputSchema = roleSchema; break;
-      case 'tenants': inputSchema = tenantSchema; break;
-      case 'account': inputSchema = accountSchema; break;
-      default: throw new Error(`Unknown resource ${resourceName}`);
+      case 'internalusers':
+        inputSchema = internalUserSchema;
+        break;
+      case 'actiongroups':
+        inputSchema = actionGroupSchema;
+        break;
+      case 'rolesmapping':
+        inputSchema = roleMappingSchema;
+        break;
+      case 'roles':
+        inputSchema = roleSchema;
+        break;
+      case 'tenants':
+        inputSchema = tenantSchema;
+        break;
+      case 'account':
+        inputSchema = accountSchema;
+        break;
+      default:
+        throw new Error(`Unknown resource ${resourceName}`);
```

Comment:
I think use a dict like below is more readable.
~~~
schemaMap = {
  "internalusers": internalUserSchema, 
  "actiongroups": actionGroupSchema,
  ...
}
inputSchema = schemaMap[resourceName]
if (inputSchema == undefined)
  throw
~~~

Reply:
updated to use schema map